client_unix.go

```diff
@@ -0,0 +1,17 @@
+package containerd
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+)
+
+func dialer(address string, timeout time.Duration) (net.Conn, error) {
+	address = strings.TrimPrefix(address, "unix://")
```

Comment:
Why validate first? Just trim it.  same reason you don't stat a file before opening, just open it